kong/db/schema/entities/routes.lua

```diff
@@ -73,6 +90,55 @@ if kong_router_flavor == "expressions" then
 
 -- router_flavor in ('traditional_compatible', 'traditional')
 else
+  local DEPRECATION_MSG
+
+  if kong_router_flavor == "traditional" then
+    DEPRECATION_MSG = "path_handling='v1' is deprecated and will be removed in future version, " ..
+                      "please use path_handling='v0' instead"
+
+  elseif kong_router_flavor == "traditional_compatible" then
+    DEPRECATION_MSG = "path_handling='v1' is deprecated and will not work under traditional_compatible " ..
+                      "router_flavor, please use path_handling='v0' instead"
+  end
+
```

Comment:
This does not seem as straightforward as the original code: is it already deprecated if we are using traditional?

Please consider renaming the `DEPRECATION_MSG` to a more specific name, like `PATH_V1_DEPRECATION_MSG`.

Reply:
done